flink-connector-jdbc/pom.xml

```diff
@@ -38,10 +38,12 @@ under the License.
 		<scala.binary.version>2.12</scala.binary.version>
 		<scala-library.version>2.12.7</scala-library.version>
 		<assertj.version>3.23.1</assertj.version>
+		<jackson.version>2.15.2</jackson.version>
 		<postgres.version>42.5.1</postgres.version>
 		<oracle.version>21.8.0.0</oracle.version>
 		<trino.version>418</trino.version>
 		<byte-buddy.version>1.12.10</byte-buddy.version>
+		<elasticsearch.version>8.8.1</elasticsearch.version>
```

Comment:
why this version? last version is 8.11.1

Reply:
Version updated.